csrc/flash_attn/src/flash_bwd_launch_template.h

```diff
@@ -18,20 +18,20 @@ __global__ void flash_bwd_clear_dkvaccum_kernel(Flash_bwd_params params) {
     flash::clear_dKVaccum<Kernel_traits>(params);
 }
 
-template<typename Kernel_traits, bool Is_dropout, bool Is_causal, bool Is_even_M, bool Is_even_K>
+template<typename Kernel_traits, bool Is_dropout, bool Is_causal, bool Is_alibi, bool Is_even_M, bool Is_even_K>
```

Comment:
Let's call it "Has_alibi" instead of "Is_alibi"? Since this stands for "has attention linear bias".